### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by EPMatt** [TheFireCo/genkit-plugins#203] [Announcements]
## What's Changed
* (feat) added support for latest sonnet 3.5 and haiku 3.5 by @danwritecode in https://github.com/TheFireCo/genkit-plugins/pull/184
* Automated Release Drafter with Version Bump PR Labels  by @EPMatt in https://github.com/TheFireCo/genkit-plugins/pull/182
* docs: add xavidop as a contributor for code, doc, and test by @allcontributors in https://github.com/TheFireCo/genkit-plugins/pull/195
* feat: azure-openai plugin by @xavidop in https://github.com/TheFireCo/genkit-plugins/pull/194
* chore(deps-dev): bump @docusaurus/types from 3.3.2 to 3.6.3 by @dependabot in https://github.com/TheFireCo/genkit-plugins/pull/177

## New Contributors
* @danwritecode made their first contribution in https://github.com/TheFireCo/genkit-plugins/pull/184

**Full Changelog**: https://github.com/TheFireCo/genkit-plugins/compare/v0.11.1...v0.12.0

<hr /><em>This discussion was created from the release <a href='https://github.com/TheFireCo/genkit-plugins/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by danwritecode** [comment]
thanks guys!
